apps/academy-wep/components/CredentialCard.tsx

```
@@ -0,0 +1,76 @@
+						<h2 className="bg-gradient-to-r from-[#000124] to-[#7CC635] bg-clip-text text-transparent font-bold text-lg sm:text-xl mb-2 sm:mb-3 text-center sm:text-left">
+							{title}
+						</h2>
+						<p className="text-[#4B5563] text-sm sm:text-base text-center sm:text-left">
+							{description}
+						</p>
+
+						{/* features */}
+						<ul className="flex flex-col sm:flex-row gap-3 sm:gap-4 md:gap-6 lg:gap-10 mt-4 sm:mt-5 text-xs list-none p-0">
+							{features.map((feature, index) => (
+								<li
+									key={index}
```

Biome (1.9.4)

[error] Avoid using the index of an array as key property in an element.

This is the source of the key value.

The order of the items may change, and this also affects performances and component state.
Check the React documentation. 

(lint/suspicious/noArrayIndexKey)